packages/mdl-typography/README.md

```diff
@@ -0,0 +1,134 @@
+# mdl-typography
+
+MDL typography is a CSS-only component that implements the
+[Material Design typography guidelines](https://material.google.com/style/typography.html), and makes them available to
+developers as CSS classes.
+
+## Installation
+
+> Note: Installation via the npm registry will be available after alpha.
+
+
+## Usage
```

Comment:
As far as docs go in V2, we should drop "Usage" tutorials. In this case, we need to alert people they will need Roboto for the font face. This will keep our docs concise to what we provide and not repeat basic web development to users.

In the end, more examples of different use-cases > holding people's hands on structuring the markup.